src/AppKit/NSFont.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Runtime.InteropServices;
+using XamCore.CoreFoundation;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.CoreGraphics;
+using XamCore.CoreAnimation;
+using XamCore.CoreText;
+
+using CGGlyph = System.UInt16;
+
+namespace XamCore.AppKit {
+	public partial class NSFont
+	{
+		public static NSFont FromCTFont (CTFont font)
+		{
+			return new NSFont (font.Handle);
+		}
+
+		[Mac (10,13)]
+		public unsafe CGRect [] GetBoundingRects (CGGlyph [] glyphs)
+		{
+			if (glyphs == null)
+				throw new ArgumentNullException ("glyphs");
+			if (glyphs.Length < 1)
+				throw new ArgumentException ("glyphs array is empty");
+
+			CGRect [] bounds = new CGRect [glyphs.Length]; 
+			fixed (CGRect* boundsPtr = &bounds [0])
```

Comment:
I used the methods in NSBezierPath as a template, but please review this pointer work.